packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.controller.js

```diff
@@ -62,25 +62,34 @@ angular.module('App').controller(
         id: value,
         text: value,
       }));
-      if (field.key === 'autocommit') {
-        PrivateDatabaseConfigurationsCtrl.convertFieldAsToggle(field);
-      } else if (field.key === 'event_scheduler') {
-        PrivateDatabaseConfigurationsCtrl.convertFieldAsToggle(field);
+
+      if (field.key === 'sql_mode') {
```

Comment:
NITPICK: Not a blocker
Is it possible to move field.key, field.type and other constants into a separate constant file? or create a ENUM?

Reply:
This code is pretty old and should be reworked at some point. But right now we don't want to do that.